### DISCUSSION TITLE
博客换了字体后，waline中文显示乱码。请问怎么解决 || After changing the font of the blog, waline Chinese displays garbled characters. How to solve it?

### DISCUSSION THREAD

**Discussion by flight025** [walinejs/waline#2202] [Q&A] [answered]
### Describe the bug | 问题描述

我的博客基于hugo,换了字体后，waline评论能够用，但是字体乱码，请问是什么原因导致的，是不是字体原因？
![乱码](https://github.com/walinejs/waline/assets/65944944/56337b8c-381c-4460-976d-973b0b3dd446)

### Website URL | 问题网站

www.zaqizaba.xyz

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

My blog is based on hugo. After changing the font, waline comments can be used, but the font is garbled. What is the cause? Is it the font?
![garbled](https://github.com/walinejs/waline/assets/65944944/56337b8c-381c-4460-976d-973b0b3dd446)

### Website URL | Problem website

www.zaqizaba.xyz

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by Mister-Hope** [comment]
![image](https://github.com/walinejs/waline/assets/33315834/dad5c31c-1a23-433f-a513-114cabdc8140)

不能复现

---
![image](https://github.com/walinejs/waline/assets/33315834/dad5c31c-1a23-433f-a513-114cabdc8140)

Cannot reproduce

**Reply by flight025** [reply, marked as answer]
奇怪。为什么我的网站我自己打开评论显示乱码？请问开发者大概知道原因是什么吗。。。

---
strangeness. Why does my website display garbled characters when I open the comments myself? Does the developer know what the reason is? . .

**Reply by Mister-Hope** [reply]
我咋知道啊，你看看你有没有自定义字体？

---
How do I know? Can you see if you have a custom font?
